src/covenantV1/bridge.ts

```diff
@@ -84,12 +90,20 @@ export function sendTransaction(
     throw new Error("Minimum fee must be bigger than 0");
   }
 
+  if (minimumFee < minBtc) {
```

Comment:
According to the audit requirements, this difference must be less than or equal to BTC_DUST_SAT